doc/admin/deployment.md

```diff
@@ -759,14 +758,13 @@ journalctl --unit daos-agent
 
 ## System Validation
 
-To validate that the DAOS system is properly installed, the daos\_test
+To validate that the DAOS system is properly installed, the daos_test
 suite can be executed. Ensure the DAOS Agent is configured and running before
-running daos\_test and that the DAOS_SINGLETON_CLI and CRT_ATTACH_INFO_PATH
-environment variables are properly set:
+running daos_test and that the DAOS_SINGLETON_CLI and CRT_ATTACH_INFO_PATH
+environment variables are properly set as described [here](./#server-startup).
```

Comment:
This link seems broken. When I click on it in Firefox, it takes me to https://github.com/daos-stack/daos/blob/master/doc/admin/#server-startup, but I assume it's supposed to be https://github.com/daos-stack/daos/blob/master/doc/admin/deployment.md#server-startup

Maybe you need to either drop the path part of the link or else add the filename? Not that familiar with markdown directives.

Reply:
that's strange, worked for me with the VS code Markdown preview plug-in